draft-ietf-rats-reference-interaction-models.md

```diff
@@ -567,7 +567,8 @@ In the Uni-Directional model, handles are composed of cryptographically signed t
 The Handles are created by an external trusted third party (TTP) -- the Handle Distributor -- which includes a trustworthy source of time, and takes on the role of a Time Stamping Authority (TSA, as initially defined in {{-TSA}}).
 Timestamps created from local clocks (absolute clocks using a global timescale, as well as relative clocks, such as tick-counters) of Attesters and Verifiers MUST be cryptographically bound to fresh Handles received from the Handle Distributor.
 This binding provides a proof of synchronization that MUST be included in all produced Evidence.
-Correspondingly, conveyed Evidence in this model provides a proof that it was fresh at a certain point in time.
+Correspondingly, conveyed Evidence in this model provides a proof that it was fresh at the specific point in time when the handle was generated during the Handle Generation phase.
```

Comment:
@thomas-fossati could you double check this please and maybe add a change suggestion?

Reply:
I don't think the current phrasing is 100% correct.
This model provides proof that evidence generation _happened after_ the handle generation phase.

Reply:
yes